utils/frame/benchmarking-cli/src/pallet/writer.rs

```diff
@@ -297,7 +297,7 @@ pub fn write_results(
 
 	// New Handlebars instance with helpers.
 	let mut handlebars = handlebars::Handlebars::new();
-	handlebars.register_helper("underscore", Box::new(UnderscoreHelper));
+	handlebars.register_helper("underscoreFormat", Box::new(UnderscoreFormatHelper));
```

Comment:
You should still register both helpers, not just replace the old one. also note the name change i recommended

Reply:
I am currently fixing the READMEs :smile:   
Regarding the other templates; they already use the correct unit since they use nano seconds before putting it into the template, for example here:  
```pre
pub const {{long_name}}Weight: Weight = {{underscore weight}} * WEIGHT_PER_NANOS;
```

Reply:
@hzy1919 just because we dont use it in this particular template, does not mean it is useless. People can provide their own templates, where they have sub-nanosecond weights, in which case they would probably want to use `underscore`

Reply:
Right. I hadn't thought of that before .